wire-ios/Wire-iOS/Sources/UserInterface/Settings/DebugActions.swift

```diff
@@ -242,8 +242,10 @@ enum DebugActions {
             switch command {
             case .repairInvalidAccessRoles:
                 DebugActions.updateInvalidAccessRoles()
-            }
 
+            case .resyncResources:
+                DebugActions.triggerResyncResources()
+            }
```

Comment:
can we expose slow sync? I guess it's not strictly needed

Reply:
if it's not needed maybe we skip it for now and revisit in the next version?